pkg/guacanalytics/patchPlanning.go

```diff
@@ -230,6 +234,18 @@ func exploreHasSourceAtFromSource(ctx context.Context, gqlClient graphql.Client,
 	return nil
 }
 
+// TODO: Expand to not just deal with packageVersions
+func explorePkgEqual(ctx context.Context, gqlClient graphql.Client, q *queueValues, pkgEqual model.NeighborsNeighborsPkgEqual) {
+	// Step 1: Loop through all the packages listed in the pkgEqual
+	// Step 2: as long as they are not equal to the current node, add them and their packageName to the queue
```

Comment:
do we need to add packageVersion to queue as well?

Reply:
Yes, good point.  Will change that.